codebuild/release/release-staging.yml

```diff
@@ -0,0 +1,92 @@
+version: 0.2
+# 'release-staging' is an AWS::CodeBuild::Project that Stages a release on CodeArtifact and validates the staging using 'validate-staging'
+# Command to trigger this codebuild from an authorized command line.
+# aws codebuild start-build-batch \
+#         --region us-west-2 \
+#         --project-name java-ddb-ec-test-release \
+#         --environment-variables-override name=VERSION,value=<INSERT_VERSION>,type=PLAINTEXT name=COMMIT_ID,value=<INSERT_COMMIT_ID>,type=PLAINTEXT
+
+env:
+  variables:
+    REGION: us-east-1
+    DOMAIN: crypto-tools-internal
+    REPOSITORY: java-ddbec-staging
+  parameter-store:
+    ACCOUNT: /CodeBuild/AccountId
+  secrets-manager:
+    GPG_KEY: Maven-GPG-Keys-Credentials:Keyname
+    GPG_PASS: Maven-GPG-Keys-Credentials:Passphrase
+
+phases:
+  install:
+    runtime-versions:
+      java: openjdk8
+  pre_build:
+    commands:
+      - git checkout $COMMIT_ID
+      - FOUND_VERSION=$(grep version pom.xml | head -n 2 | sed -n 's/[ \t]*<version>\(.*\)<\/version>/\1/p')
```

Comment:
So the DDBEC is structured a bit weirdly in that the version we want to actually check is at `sdk1/pom.xml`

There is a top level pom (it's intention was to de-dupe configuration between sdk1 and sdk2 poms when that was under development. The in progress sdk2 work was ripped out, but the pom structure stayed.), but it never gets packaged itself as a jar and released.